tests/widget/modal/widget-modal.configurable-journey.test.js

```diff
@@ -23,10 +23,12 @@ test('modal widget and testing changing journeys', async ({ page }) => {
 
   // Change to go back to Sign In
   const backToSignInBtn = page.getByRole('link', { name: 'Sign in here!' });
+  await backToSignInBtn.waitFor({ state: 'visible' });
```

Comment:
I'm really torn on this. In researching this, I believe any element we get and then apply some sort of action to it, like this:

```
const backToSignInBtn = page.getByRole('link', { name: 'Sign in here!' });
await backToSignInBtn.click(); <-- this should internally have a `waitFor` built into it
```

... has a `waitFor` built into it, so adding a `waitFor` duplicates this and is a bit misleading. I get why we want to add it, but let's keep iterating on this and make sure we really understand what we are adding here. I just say this in order to keep our tests from getting bloated without any real value being delivered to offset the cost.

If stuff is timing out because of the server, we _can_ add a custom timeout to the element query itself.